packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -15,10 +15,7 @@ import { applyMixins } from "../utilities/apply-mixins.js";
  * determines if element is an HTMLElement and if it has the role treeitem
  */
 export function isTreeItemElement(el: Element): el is HTMLElement {
-    return (
-        isHTMLElement(el) &&
-        (el.getAttribute("role") === "treeitem" || el.tagName.includes("TREE-ITEM"))
-    );
+    return isHTMLElement(el) && el.hasOwnProperty("isTreeItem");
```

Comment:
It just occurred to me that using `hasOwnProperty` may be an issue here since that only checks for the property immediately on self not accounting for inheritance. But, we do want to account for inheritance. So, I think we want something like this:

```ts
return isHTMLElement(el) && (el as any).isTreeItem;
```

Reply:
Good catch - updated.